frontend/src/uxbox/main/ui/utils.cljs

```diff
@@ -0,0 +1,20 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
+
+(ns uxbox.main.ui.utils
+  (:require
+   [rumext.alpha :as mf]
+   [beicon.core :as rx]))
+
+(defn use-stream
```

Comment:
I think we can add this function to uxbox.main.ui.hooks ns that already has many of them.

Reply:
You're right